thermalnetwork/network.py

```diff
@@ -270,7 +272,15 @@ def set_components(self, comp_data_list: list[dict], throw: bool = True):
         self.components_data.append(obj)
 
         # Add WAHP
```

Comment:
@mitchute is `WAHP` WAter Heat Pump?

Reply:
Usually Water-to-air heat pump. WWHP would be water-to-water.